packages/components/src/internal/components/forms/input/RadioGroupInput.tsx

```diff
@@ -13,6 +13,50 @@ export interface RadioGroupOption {
     value: string;
 }
 
+interface RadioGroupOptionImplProps {
+    isSelected: boolean;
+    name: string;
+    onSetValue: (value: string) => void;
+    option: RadioGroupOption;
+    showDescriptions: boolean;
+}
+
+const RadioGroupOptionImpl: FC<RadioGroupOptionImplProps> = memo(props => {
```

Comment:
nit: This can be named `RadioGroupOption`
nit: This needs a `displayName`

Reply:
done